src/components/Picker/index.android.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import RNPickerSelect from 'react-native-picker-select';
+
+import * as pickerPropTypes from './pickerPropTypes';
+import styles from '../../styles/styles';
+
+const Picker = ({
+    onChange,
+    items,
+    useDisabledStyles,
+    placeholder,
+    value,
+    icon,
+}) => (
+    <RNPickerSelect
+        onValueChange={onChange}
+        items={items}
+        style={useDisabledStyles ? {
+            ...styles.picker,
+            inputAndroid: [
+                styles.picker.inputAndroid, styles.textInput, styles.disabledTextInput,
+            ],
```

Comment:
not sure how platform specific styles are handled but would it make more sense to have the ios/android/index files just export the style and the component itself be defined once? would avoid duplication and make it easier to update without having to touch 3 files